CS361Group3Project/templates/login/login.html

```diff
@@ -0,0 +1,26 @@
+{% extends 'base.html' %}
+
+{% block content %}
+<div>
+  <h1>Sign In</h1>
+  <div class="row">
+    <form action="/" method="post" class="form">
+      {% csrf_token %}
+      <div class="row">
+        <div class="col">
+            <div class="mb-3">
+                <label class="form-label" for="username">Username</label>
+                <input type="text" name="username" class="form-control" id="username">
```

Comment:
fyi you can use the required tag to prevent a form from being submitted if the field is empty.
Not important, just observation